docs/iidm/importer/iidm.md

```diff
@@ -66,7 +66,7 @@ The IIDM importer has three importing modes:
 </iidm:network>
 ```
 
-# Configuration properties for IIDM-XML importer
+# Configuration properties for the XIIDM importer
```

Comment:
I prefer previous formula.

Reply:
OK, I disagree but did what you asked.To be coherent, I also changed docs/iidm/export/iidm.md to use IIDM-XML as well.

For info, XIIDM is used 30 times in our docs, accros all the documents. IIDM-XML is used ony in these two files, 7 times.

The itools  command line  arguments are also --output-format XIIDM

We should work hard to have more coherent naming but this can be done in another PR

Reply:
I think both are OK: XIIDM is an acronym for IIDM in XML. In the next version of the converter, we'll probably present things like this... And one day, when the JIIDM will be ready it will make sense

Reply:
I agree with the fact that we need to improve the naming in the whole web site, probably in a separate PR. For XIIDM and for JIIDM, I prefer a more explicit name like IIDM XML and IIDM JSON.

Reply:
I like XIIDM but I don't like JIIDM (people will think "java iidm") :(